nix/cells/plutus/library/make-plutus-project.nix

```diff
@@ -156,6 +156,7 @@ let
           cardano-crypto-class.components.library.pkgconfig = lib.mkForce [
             [ pkgs.libsodium-vrf pkgs.secp256k1 ]
           ];
+          cardano-crypto-class.components.library.libs = lib.mkForce [ pkgs.blst ];
```

Comment:
This gives you warnings about a missing dynamic library, which is misleading because it's a static library and it isn't missing. I've asked nix experts and they think there might be something genuinely wrong here.  It's not clear how to resolve this: maybe `blst` needs a `pkg-config` file somewhere (not something we can do in the plutus repository)/

Reply:
> This does nothing at the moment, right? Since we're not using the code from cardano-crypto-class?

No, this is for the `libblst.a` C library, which needs to be available at link time, like `libsodium` and the like.

Reply:
pkg-config logic should be in the most recent iohk-nix.